### PR TITLE
Add liquidity pools to DAGs

### PR DESCRIPTION
**Pull request by sydneynotthecity**
The `liquidity_pools` tables can be populated through either the use of the `bounded_core` DAG or the `bucket_list` DAG (used for initial table hydration).

### CODE REVIEW

File: dags/stellar_etl_airflow/build_load_task.py [hunk 1]
```diff
@@ -111,7 +111,10 @@ def upload_to_gcs(data_type, prev_task_id, batch_stats, **kwargs):
     filename = kwargs['task_instance'].xcom_pull(task_ids=prev_task_id)
     logging.info(type(filename))
```
Comment (erika-sdf) [inline dags/stellar_etl_airflow/build_load_task.py:112]: This looks like something leftover from debugging. Should we remove it?

File: dags/stellar_etl_airflow/build_load_task.py [hunk 2]
```diff
@@ -111,7 +111,10 @@ def upload_to_gcs(data_type, prev_task_id, batch_stats, **kwargs):
...
     if isinstance(filename, dict):
-        filename = filename["output_file"]
+        if 'start' in filename.keys():
+            filename = f'{filename["start"]}-{filename["start"]+63}-{data_type}.txt'
```
Comment (erika-sdf) [inline dags/stellar_etl_airflow/build_load_task.py:115]: nit: Can this entire line be pulled into a function that builds the filename?

**Review by erika-sdf — APPROVED** [review]
A small nit, but otherwise LGTM.

File: dags/stellar_etl_airflow/build_load_task.py (hunk 2 above)
Comment (sydneynotthecity) [reply dags/stellar_etl_airflow/build_load_task.py]: Done!

**Pull request merged**